### ISSUE TITLE
Django 3.0 model choices missing label type

### ISSUE THREAD

**Issue by 09F911029D74E35BD84156C5635688C0** [typeddjango/django-stubs#346]
# Bug report

`Choices.label` is missing from the base choices classes

```python
from django.db.models import IntegerChoices

class Foo(IntegerChoices):
    BAR = 1, "Bar"

print(Foo.BAR.label)
print(Foo.BAR.name)
print(Foo.BAR.value)
```

Runs fine:

```
Bar
BAR
1
```

## What's wrong

Unexpected error

```python
error: "Foo" has no attribute "label"
```

## How is that should be

Expecting no issues. Type should be `str`, other types rejected by stdlib `Enum`.

## System information

- OS: linux
- `python` version: 3.8
- `django` version: 3.0
- `mypy` version: 0.770
- `django-stubs` version: 1.5.0

**Comment by mlissner** [comment]
I'm having trouble with IntegerChoices too, due to this code:

```
from http import HTTPStatus
HttpStatusCodes:  = models.IntegerChoices(
    "HttpStatusCodes", [(s.name, s.value) for s in HTTPStatus]
)
```

When I run mypy on this, I get:

```txt
cl/api/models.py:25: error: No overload variant of "IntegerChoices" matches argument types "str", "List[Tuple[str, int]]"
cl/api/models.py:25: note: Possible overload variant:
cl/api/models.py:25: note:     def IntegerChoices(cls, x: Union[str, bytes], base: int) -> IntegerChoices
cl/api/models.py:25: note:     <1 more non-matching overload not shown>
```

**Comment by sobolevn** [comment]
PRs are welcome! 👍

**Comment by kevinmarsh** [comment]
Think the original issue was solved in #406, the follow on comment about `No overload variant` is still a problem which I've just run into, but suggest that gets [it's own issue](https://github.com/typeddjango/django-stubs/issues/729) since it's related to [enum's functional API](https://docs.python.org/3/library/enum.html#functional-api)

**Comment by flaeppe** [comment]
Closed via #406 

Agreed on the own issue regarding `No overload variant`
